pkg/gitreceive/build_test.go

```diff
@@ -0,0 +1,87 @@
+package gitreceive
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/arschles/assert"
+	"github.com/deis/builder/pkg"
+	"github.com/deis/builder/pkg/gitreceive/storage"
+)
+
+const (
+	bucketName = "mybucket"
+	objKey     = "myobj"
+)
+
+func TestGetProcFileFromRepoSuccess(t *testing.T) {
+	tmpDir := os.TempDir()
+	data := []byte("web: example-go")
+	if err := ioutil.WriteFile(tmpDir+"/Procfile", data, 0644); err != nil {
+		t.Fatalf("error creating %s/Procfile (%s)", tmpDir, err)
+	}
+	defer func() {
+		if err := os.Remove(tmpDir + "/Procfile"); err != nil {
```

Comment:
this is gonna leak the actual temporary directory itself. can you change this to `os.RemoveAll(tmpDir)` so that the dir also is removed? Same with the below test